test/integration/metrics_test.go

```diff
@@ -146,7 +146,9 @@ func TestHighCPU(t *testing.T) {
 			assert.True(t, cpu.KernelPercent >= 0,
 				"kernel percent not >= 0")
 
-			assert.Truef(t, cpu.UserPercent+cpu.KernelPercent <= cpu.CPUPercent,
+			// This test is flaky, the +2 should not be needed, but we noticed that from time to time due to race conditions,
+			// such value is slightly higher
+			assert.Truef(t, cpu.UserPercent+cpu.KernelPercent <= cpu.CPUPercent+1,
```

Comment:
I think you mean `+1` in the comment. We could also use [assert.inDelta](https://pkg.go.dev/github.com/stretchr/testify/assert#InDelta)

Reply:
Nice!